src/Main.hs

```diff
@@ -1,5 +1,10 @@
+{-# LANGUAGE OverloadedStrings #-}
+
 import Options.Applicative
 import Data.Char (isSpace)
+import Data.Text (Text)
+import qualified Data.Text as T
+import qualified Data.Text.IO as T (readFile, getContents)
```

Comment:
I think we decided in a previous discussion slack that qualified with limited imports isn't valuable / doesn't make sense.